test/e2e/functional_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build functional
-// +build functional
```

Comment:
functional e2e keeps growing.  Most of the time it is timeouts. This PR splits core functional testcases and other functional testcase in separate steps in CI. This will improve the e2e test duration also.